blog/2023-06-16-inline-sdk.mdx

```diff
@@ -0,0 +1,1104 @@
+---
+title: Creating Custom Bands with the Inline SDK
+author: Michael Kowal
+authorTitle: Software Developer
+authorURL: https://github.com/MichaelKowal
+authorImageURL: https://github.com/MichaelKowal.png
+tags: [inline, web]
+---
+
+import Link from "@docusaurus/Link";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+With VertiGIS Inline `2.3` and later, users now have the ability to create their own custom bands that run right in Inline using the VertiGIS Inline SDK. These custom bands are made with React and Typescript, giving authors total control over what the band can do. Today, we will run through the steps to build, deploy and use a custom band in an Inline view.
+
+<!--truncate-->
+
+:::note
+To use these new features, you will need the latest versions of VertiGIS Inline, VertiGIS Studio Web, and the VertiGIS Inline SDK.
+:::
+
+## Getting Started
+
+You will need a few tools to build a custom band:
+
+-   [Download and install node](https://nodejs.org/en)
```

Comment:
I would go with the full names:
- Node.js
- Visual Studio Code

Reply:
Fixed